MailCore/Utils/Constants.swift

```diff
@@ -224,4 +224,15 @@ public enum Constants {
             return dateFormatter.date(from: date)
         }
     }
+
+    public static var canOSBeUpdated: Bool {
```

Comment:
After a second thought, something like isUsingABreakableOSVersion would be better